adapters/Unity/UnityAdapter/GADMUnityInterstitialAd.h

```diff
@@ -20,7 +20,7 @@
 @protocol GADMAdNetworkAdapter;
 @protocol GADMAdNetworkConnector;
 
-@interface GADMUnityInterstitialAd : NSObject <UnityAdsExtendedDelegate, UnityAdsLoadDelegate>
```

Comment:
Just curious here, what happened to the `UnityAdsExtendedDelegate`?

Reply:
The functionality was moved to the UnityAdsShowDelegate.  Between UnityAdsLoadDelegate and UnityAdsShowDelegate, I don't think the extended delegate is needed anymore.

Reply:
I did not remove the functions implementing the UnityAdsExtendedDelegate for now (even though they are never called) so they can explain where the functionality has moved to.

Reply:
can we put a blocker or smth that will help us debug in case ExtendedDelegate function are called. 
or maybe we can remove ExtendedDelegate if its not used anymore? What do you think?

Reply:
Personally I would prefer to just remove all the unused methods.  I did it this way because I seem to remember pushback from the mediators in the past because they didn't like things changing too quickly and seemed to prefer leaving the old methods in with a comment to explain where the logic moved.

I assumed we would remove the functions at a future time (for example the 4.0 adapter update), but maybe it would be better to remove them now.  If nobody complains, the code will be cleaner, otherwise, I can just add it back in with the blocker/smth.

Reply:
Gotcha. 👍